### PR TITLE
add Qalculate! extension

### PR DESCRIPTION
**Pull request by XDjackieXD**
This adds basic Qalculate support (see albertlauncher/albert#633).
There are still things that could be done like switching output format by invoking the plugin with "hex=" or "bin=" instead of just "=" but basic functionality is working.

### CODE REVIEW

**Comment by ManuelSchneid3r** [comment]
Okay this is cool, but it need a lot of more work.
libqalculate is pretty comlex.

File: q_calculator/FindQalculate.cmake [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      NAMES
+      libqalculate
+      PATHS
+      ${_pc_QALCULATE_INCLUDE_DIRS}
+      ${INCLUDE_INSTALL_DIR}
+    )
+
+    if(_pc_QALCULATE_FOUND)
+      if(${_pc_QALCULATE_VERSION} VERSION_LESS 2.0.0)
+        find_library(CLN_LIBRARIES
```
Comment (ManuelSchneid3r) [inline q_calculator/FindQalculate.cmake]: Whats this?

Comment (ManuelSchneid3r) [reply q_calculator/FindQalculate.cmake]: Nice! An arbitrary precision floating point library :D But acutally it is not used.

**Comment by ManuelSchneid3r** [comment]
is there a way to localize the calculator? I did not find any configuration for the parser. This is the first thing ppl will complain about as soon this extension is released.

**Comment by XDjackieXD** [comment]
What's the reason you disabled unkown variables completely? Only activating them when the calculator was explicitely invoked with the trigger seems like a good idea as solving equations with variables can be pretty useful.

As for why the CLN libs are also included: older versions of lobqalculate rquire them. I haven't looked if some distros still ship this old versions though so I don't know how relevant that is.

**Comment by ManuelSchneid3r** [comment]
lol

**Comment by ManuelSchneid3r** [comment]
are you in the chat?

**Comment by ManuelSchneid3r** [comment]
join https://telegram.me/albert_launcher_community

**Pull request closed without merging**